crucible-wasm/src/Lang/Crucible/Wasm/Translate.hs

```diff
@@ -622,11 +622,11 @@ genInstruction genReturn im st ctrlStack instr =
             Just (ConstantGlobal _) -> panic "genInstruction: SetGlobal" ["setGlobal of constant global"]
             Just (MutableGlobal gv) -> writeGlobal gv =<< checkStackVal (globalType gv) =<< popStack
 
-    Wasm.CurrentMemory ->
+    Wasm.MemorySize ->
```

Comment:
Should we add the following to a ticket as well so it doesn't get lost?

> I have not added support for the `memory.fill`, `memory.copy`,
  `memory.init`, or `data.drop` instructions, all of which are new in Wasm 2.0.

Reply:
I've opened https://github.com/GaloisInc/crucible/issues/1233 for this.